content/en/docs/tasks/administer-cluster/change-default-storage-class.md

```diff
@@ -96,7 +96,7 @@ for details about addon manager and how to disable individual addons.
 {{% /capture %}}
 
 {{% capture whatsnext %}}
-* Learn more about [StorageClasses](/docs/concepts/storage/persistent-volumes/).
```

Comment:
we are learning about storage classes here lets keep this as it is

Reply:
I see, so would you agree that we should at least make the link be to /docs/concepts/storage/storage-classes - since people will be clicking on "StorageClasses"?